lte/gateway/c/oai/tasks/grpc_service/AmfServiceImpl.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2020 The Magma Authors.
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include <string>
+
+#include "lte/protos/session_manager.pb.h"
+#include "lte/protos/spgw_service.pb.h"
+#include <folly/IPAddress.h>
+
+extern "C" {
+#include "spgw_service_handler.h"
```

Comment:
Why is this header needed here? If it is needed for future code, please include it in future PRs.

Reply:
yeah, this is not required, I'll remove and push the changes.